SignalMessaging/environment/AppSetup.m

```diff
@@ -50,6 +50,7 @@ + (void)setupEnvironmentWithCallMessageHandlerBlock:(CallMessageHandlerBlock)cal
         // Register renamed classes.
         [NSKeyedUnarchiver setClass:[OWSUserProfile class] forClassName:[OWSUserProfile collection]];
         [NSKeyedUnarchiver setClass:[OWSDatabaseMigration class] forClassName:[OWSDatabaseMigration collection]];
+        [NSKeyedUnarchiver setClass:[OWSContactShare class] forClassName:@"OWSContact"];
```

Comment:
Nice - forgot about this dance.

Reply:
I tried to add conversions for existing contacts, but it ended up being a nightmare. Once this is merged, you'll probably need to do clean installs on any debug devices.